core/shared/src/main/scala/matryoshka/patterns/ListF.scala

```diff
@@ -50,7 +50,7 @@ object ListF {
   def seqIso[A] = AlgebraIso[ListF[A, ?], Seq[A]] {
     case ConsF(h, t) => h +: t
     case NilF()      => Seq.empty
-  } (s => s.headOption.fold[ListF[A, Seq[A]]](NilF())(ConsF(_, s.tail)))
+  } (s => s.headOption.fold[ListF[A, Seq[A]]](NilF())(ConsF(_, s.drop(1))))
```

Comment:
Oh, this doesn’t pattern-match because it’s on `Seq`, not `List`. But, this should probably go away anyway. I’ve had a few comments that you just can’t deal with `Seq` in a reasonable way.